swagger.json

```diff
@@ -5118,6 +5229,83 @@
                 }
               
         },
+        "HosAuditLogBase": {
+            "type": "object",
+            "properties": {
+                "driverId": {
+                    "type": "integer",
+                    "description": "Driver ID",
+                    "example": 1234
+                },
+                "vehicleId": {
+                    "type": "integer",
```

Comment:
Can we add `format: int64` for these 2 integers?

Reply:
yep, this was a copy/paste from an out of date reference I think!